WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java

```diff
@@ -567,6 +569,27 @@ public void onErrorResponse(VolleyError error) {
     }
 
     @SuppressWarnings("unused")
+    public void onEventMainThread(final NotificationEvents.NoteLikeStatusChanged event) {
+        // Like/unlike done -> refresh the note and update db
+        NotificationsActions.downloadNoteAndUpdateDB(event.noteId,
+                new RestRequest.Listener() {
+                    @Override
+                    public void onResponse(JSONObject response) {
+                        EventBus.getDefault().removeStickyEvent(NotificationEvents.NoteLikeStatusChanged.class);
+                        //now re-set the object in our list adapter with the note saved in the updated DB
+                        Note note = NotificationsTable.getNoteById(event.noteId);
```

Comment:
It's unlikely that the note is null here, but better to have a check before replacing the note in the adapter.